components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -63,6 +63,9 @@ def wrapper(self, *args, **kwargs):
                 import django  # NOQA
             except:
                 self.ctx.die(681, "Django not installed!")
+            if django.VERSION < (1, 6) or django.VERSION >= (1, 9):
+                self.ctx.die(684, "Django version %s is not "
```

Comment:
My main concern is that the `die` call could be too strong here. I can easily think of developers who would like to test OMERO.web behavior on the newest version of Django and know what they are doing. Would a warning not be sufficient to alter sysadmins?